Makefile

```diff
@@ -164,24 +136,44 @@ endif
 # Build tsh against libfido2?
 # FIDO2=yes and FIDO2=static enable static libfido2 builds.
 # FIDO2=dynamic enables dynamic libfido2 builds.
-LIBFIDO2_MESSAGE := without libfido2
+LIBFIDO2_MESSAGE := without-libfido2
 ifneq (, $(filter $(FIDO2), yes static))
-LIBFIDO2_MESSAGE := with libfido2
+LIBFIDO2_MESSAGE := with-libfido2
 LIBFIDO2_BUILD_TAG := libfido2 libfido2static
 else ifeq ("$(FIDO2)", "dynamic")
-LIBFIDO2_MESSAGE := with libfido2
+LIBFIDO2_MESSAGE := with-libfido2
 LIBFIDO2_BUILD_TAG := libfido2
 endif
 
 # Enable Touch ID builds?
 # Only build if TOUCHID=yes to avoid issues when cross-compiling to 'darwin'
 # from other systems.
-TOUCHID_MESSAGE := without Touch ID
+TOUCHID_MESSAGE := without-Touch-ID
 ifeq ("$(TOUCHID)", "yes")
-TOUCHID_MESSAGE := with Touch ID
+TOUCHID_MESSAGE := with-Touch-ID
 TOUCHID_TAG := touchid
 endif
 
+# Enable libpcsclite for testing?
+# Eargerly enable if we detect the package, we want to test as much as possible.
+ifeq ("$(shell pkg-config libpcsclite 2>/dev/null; echo $$?)", "0")
+LIBPCSCLITE_TEST_TAG := libpcsclite
+endif
+
+# Build teleport/api against libsclite?
+# Only build if LIBPCSCLITE=yes.
+# This is used for PIV functionality.
+LIBPCSCLITE_MESSAGE := without-PIV-support
+ifneq (, $(filter $(LIBPCSCLITE), yes static dynamic))
```

Comment:
Why are you checking the message here? You could check the `LIBPCSCLITE_BUILD_TAG` here and check if it's set already.